src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/TableType.java

```diff
@@ -13,13 +13,28 @@
 import org.sagebionetworks.repo.model.table.SubmissionView;
 import org.sagebionetworks.repo.model.table.TableEntity;
 import org.sagebionetworks.repo.model.table.ViewTypeMask;
+import org.sagebionetworks.web.client.DisplayConstants;
+
 
 public enum TableType {
-	table(null), submission_view(null), projects(PROJECT), tables(TABLE), folders(FOLDER), folders_tables(FOLDER | TABLE), files(FILE), files_tables(FILE | TABLE), files_folders(FILE | FOLDER), files_folders_tables(FILE | FOLDER | TABLE), dataset(FILE);
+	table(null, DisplayConstants.TABLE),
+	submission_view(null, DisplayConstants.SUBMISSION_VIEW),
+	projects(PROJECT, DisplayConstants.PROJECT_VIEW),
+	tables(TABLE, DisplayConstants.VIEW),
+	folders(FOLDER, DisplayConstants.VIEW),
+	folders_tables(FOLDER | TABLE, DisplayConstants.VIEW),
+	files(FILE, DisplayConstants.FILE_VIEW),
+	files_tables(FILE | TABLE, DisplayConstants.VIEW),
+	files_folders(FILE | FOLDER, DisplayConstants.VIEW),
+	files_folders_tables(FILE | FOLDER | TABLE, DisplayConstants.VIEW),
+	dataset(FILE, DisplayConstants.DATASET);
+
```

Comment:
Trying to come up with a reliable way to map each table type to what we typically call it.

In short:
TableEntity -> "Table"
SubmissionView -> "Submission View"
Dataset -> "Dataset"
EntityView with only files in mask -> "File View"
EntityView with only projects in mask -> "Project View"
EntityView with any other mask -> "View"

Did I get this right?

Reply:
LGTM